packages/protocol-kit/tests/e2e/offChainSignatures.test.ts

```diff
@@ -300,37 +261,11 @@ describe('Off-chain signatures', () => {
       }
     )
 
-    itif(process.env.ETH_LIB === 'web3')(
```

Comment:
If we remove this test wouldn't make sense to remove the `itif` from the previous one?